mona_openai/mona_openai.py

```diff
@@ -350,6 +367,9 @@ def get_rest_monitor(
 
     client, async_client = mona_clients_getter(mona_creds)
 
+    client.create_openai_context_class(context_class, openai_endpoint_name)
```

Comment:
Not sure I understand what is the difference between this line and the next one? Maybe add some short doc to explain what is the purpose of each one?

Reply:
Another good catch. Removed.